enterprise/frontend/src/embedding-sdk-bundle/types/plugins.ts

```diff
@@ -29,7 +29,9 @@ export type MetabaseDataPointObject = {
 export type MetabaseClickActionPluginsConfig = (
   clickActions: MetabaseClickAction[],
   clickedDataPoint: MetabaseDataPointObject,
-) => MetabaseClickAction[] | MetabaseClickAction;
+) =>
+  | MetabaseClickAction[]
+  | { onClick: (context: { closePopover: () => void }) => void };
```

Comment:
Why is the `closePopover` needed if it doesn't show the popover in the first place? I thought that if we return the single object form, we do not show the popover.